Remove references to SCA Part 2 (#2909)
theforeman/foreman-documentation: guides/common/modules/proc_creating-an-activation-key.adoc

[id="Creating_an_Activation_Key_{context}"]

> = Creating an activation key

Reviewer: As mentioned in https://github.com/theforeman/foreman-documentation/pull/2745#discussion_r1504337915 , should add a blurb here like

```
Create an activation key to assign various attributes to hosts at registration time.
```

Activation key attaches subscriptions automatically.

. Activation key providing a custom set of subscription for auto-attach.
If there are subscriptions specified and auto-attach is enabled, hosts using the activation key select the best fitting subscription from the list specified in the activation key.
Setting system purpose on the activation key does not affect this scenario.
. Activation key with the exact set of subscriptions.
If there are subscriptions specified and auto-attach is disabled, hosts using the activation key are associated with all subscriptions specified in the activation key.
Setting system purpose on the activation key does not affect this scenario.

.Custom Products

> If a {customproduct}, typically containing content not provided by Red Hat, is assigned to an activation key, this product is always enabled for the registered content host regardless of the auto-attach setting.

Reviewer: Remove all of this (lines 4-16, from "Activation key attaches subscriptions automatically" to and including "If a {customproduct}..."

[id="Creating_an_Activation_Key_{context}"]
= Creating an activation key

> Create an activation key to assign various attributes to hosts during registration.

Reviewer: This wording works for me too :+1:

. From the *Environment* list, select the environment to use.
. From the *Content View* list, select a content view to use.
ifndef::orcharhino[]
. In the *Repository Sets* tab, override repositories to Enabled or Disabled as desired.

> For more information, see {ContentManagementDocURL}Enabling_Red_Hat_Repositories_content-management[Enabling Red Hat repositories] in _{ContentManagementDocTitle}_.

Reviewer: Wrong kind of repository enabling (I know..)

This should instead link to https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#enabling-and-disabling-repositories-on-activation-key_content-management

. From the *Environment* list, select the environment to use.
. From the *Content View* list, select a content view to use.
ifndef::orcharhino[]
. In the *Repository Sets* tab, override repositories to Enabled or Disabled as desired.

> For more information, see {ContentManagementDocURL}enabling-and-disabling-repositories-on-activation-key_content-management[Enabling and disabling repositories on activation key] in _{ContentManagementDocTitle}_.

Reviewer 2: I this this should be an xref too.

endif::[]
. Click *Save*.
. Optional: In the *System Purpose* section, you can configure the activation key to set system purpose attributes on hosts during registration.

> This helps determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.

Reviewer 2: ```suggestion
This helps determine which repositories are available on the host.
It also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```
Or another way of splitting the long sentence.

Author: I don't see this anymore in my text editor.

> . In the *Repository Sets* tab, ensure only your named repository is enabled.

Reviewer: I feel like this was left over from when custom products were enabled by default. In any case, I think this step could be removed. Or, we could just say more generally

```
. In the *Repository Sets* tab, override repositories to Enabled or Disabled as desired. For more information, see xxx
```

endif::[]
. Click *Save*.

> . Optional: In the *System Purpose* section, you can configure the activation key with system purpose to set on hosts during registration.
> You can configure the activation key to set system purpose attributes on hosts during registration.

Reviewer: This says the same thing twice.

```suggestion
. Optional: In the *System Purpose* section, you can configure the activation key to set system purpose attributes on hosts during registration.
```

--content-view "_Stack_" \
--organization "_My_Organization_"
----
. Optional: For {RHEL} 8 hosts, enter the following command to configure the activation key with system purpose to set on hosts during registration to enhance subscriptions auto attachment.
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# hammer activation-key update \
--organization "_My_Organization_" \
--name "_My_Activation_Key_" \
--service-level "_Standard_" \
--purpose-usage "_Development/Test_" \
--purpose-role "_{RHELServer}_" \
--purpose-addons "_addons_"

> ----

Reviewer: This should be left in. The only part that's no longer accurate is "to enhance subscriptions auto attachment."

Also, system purpose is not limited to RHEL8.

Perhaps
```
Optional: Enter the following command to configure the activation key with system purpose attributes to set on hosts during registration.
This can help determine which repositories are available on the host, and also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

--content-view "_Stack_" \
--organization "_My_Organization_"
----
. Optional: Enter the following command to configure the activation key with system purpose attributes to set on hosts during registration.
This helps determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.

> +

Reviewer: Need to remove some stuff below this

```
. Obtain a list of your subscription IDs:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# hammer subscription list --organization "_My_Organization_"
----
. Attach the {RHEL} subscription UUID to the activation key:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# hammer activation-key add-subscription \
--name "_My_Activation_Key_" \
--name "_My_Activation_Key_" \
--organization "_My_Organization_"
----
```

--content-view "_Stack_" \
--organization "_My_Organization_"
----
. Optional: Enter the following command to configure the activation key with system purpose attributes to set on hosts during registration.

> This helps determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.

Reviewer 2: ```suggestion
This helps determine which repositories are available on the host.
It also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

Author: I don't see this anymore in my text editor either.

--organization "_My_Organization_"
----
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# hammer activation-key product-content \
--name "_My_Activation_Key_" \
--organization "_My_Organization_"

> ----

Reviewer: This doesn't seem to be needed. Doesn't it just repeat the command directly above it?

edit: Oh, I see the one above has `--content-access-mode-all true`. I think we do need that for SCA, so this change should be removed and we should just keep the existing one.
